doc-content/enterprise-only/fuse/kie-camel-con.adoc

```diff
@@ -0,0 +1,60 @@
+[id='kie-camel-con_{context}']
+= The `kie-camel` component
+
+The `kie-camel` component is an Apache Camel endpoint provided by {FUSE_LONG} that integrates {FUSE} with {PRODUCT}.  It enables you to specify a {PRODUCT} module by using a Maven group ID, artifact, and version (GAV)  which you can pull into the route and execute. It also enables you to specify portions of the message body as facts.You can use the `kie-camel` component with embedded engines or with {KIE_SERVER}.
+
+.Embedded engines
+In this scenario, KIE engines run in the same container as the {FUSE} integration project and you can communicate with engines using Kie commands. To create the Camel producer, use following URI:
+[source]
+----
+kie-local:kie-session-name?action=execute
+----
+
+For example, enter the following command to initialize a Camel route in Spring:
+
+[source]
+----
+<from uri="direct:runCommand" />
+  	<to uri="kie-local:kie-session1?action=execute"/>
+----
+
+.{KIE_SERVER}
+In this scenario, the `kie-camel` component connects to {KIE_SERVER} using the {KIE_SERVER} REST API. The enables users to communicate with the {KIE_SERVER} using the {KIE_SERVER} API. To create a producer, use following URI:
+
+[source]
+----
+kie:http://username:password@kie-server-url`
+----
+
+For example, enter the following command to initialize a Camel route in Spring:
+
+[source]
+----
+<from uri="direct:runCommand" />
+  	<to uri="kie:http://user:psswd@localhost:8080/kie-server-services/services/rest/server"/>
+----
+
+The message has the following headers:
+
+.Message headers and descriptions
+[cols="1,1", frame="all", options="header"]
+|===
+| Header
+| Description
+
+|
+|
+
+|`CamelKieClient`
+| Execution server client (mandatory)
```

Comment:
For each, replace "Execution server" with {KIE_SERVER}. Execution server is what KIE Server functions as in the product, but it's name is KIE Server. So it's correct as is, but not as clear.